packages/next/shared/lib/loadable.js

```diff
@@ -71,8 +71,14 @@ function createLoadableComponent(loadFn, options) {
     options
   )
 
+  let lazyComponent
   if (opts.suspense) {
-    opts.lazy = React.lazy(opts.loader)
+    lazyComponent = React.lazy(opts.loader)
+    return LazyImpl
```

Comment:
feel like we don't really need this option in `next/dynamic` if it only supports concurrent mode? `dynamic(loader, {suspense: true})` equals `React.lazy(loader)`. thoughts? @shuding @devknoll